addons/stock/views/stock_quant_views.xml

```diff
@@ -282,7 +282,7 @@
                             <field name="package_type_id"/>
                             <field name='company_id' groups="base.group_multi_company"/>
                             <field name='owner_id' groups="stock.group_tracking_owner"/>
-                            <field name="location_id" options="{'no_create': True}"/>
+                            <field name="location_id" style="width:50%" options="{'no_create': True}"/>
```

Comment:
Why the `width: 50%` ?
I saw nothing in the spec about that and I don't understand why you need to apply a style like this only in one field in the group but not the others ?

Reply:
Hey, @svs-odoo It was asked in spec `1(d) img 12` and I also applied width using class instead of style.